### ISSUE TITLE
MLAPI Golden Path Foundation Module CMD keeps giving me access denied error

### ISSUE THREAD

**Issue by Briancoughlin** [Unity-Technologies/com.unity.multiplayer.docs#287]
Raised by user Qazuro in forums:

There's a part in this tutorial doc that requires me to use CMD and put some commands in it. The section is called "Testing the command line helper". I keep putting the commands in and getting an access denied response. I have checked my user permissions in Windows 10 and run CMD in administrator and not administrator. What should I try now?

Just in case I'm crazy missing something here's what it spits out

Microsoft Windows [Version 10.0.18363.1556]
(c) 2019 Microsoft Corporation. All rights reserved.

C:\Users\jawf><OneDrive/Desktop/GoldenPath>/Build/GoldenPath.exe -logfile log-server.txt -mlapi server
Access is denied.

C:\Users\jawf>

**Comment by s-omeilia-unity** [comment]
Working through today

**Comment by s-omeilia-unity** [comment]
So two things I noticed when I went through the Golden Path for Windows that remedied a similar issue for me:
1. Make sure you remove the `< >` from your command.
2. Make sure your `/` is actually `\`.

Based on what you have above, your command should look like:
`C:\Users\jawf>OneDrive\Desktop\GoldenPath\Build\GoldenPath.exe -logfile log-server.txt -mlapi server`

If you are still getting a permissions error, try this command first (it will run the client and server programs simultaneously):
`C:\Users\jawf>OneDrive\Desktop\GoldenPath\Build\GoldenPath.exe -mlapi server & OneDrive\Desktop\GoldenPath\Build\GoldenPath.exe -mlapi client'
